src/lib/init.ts

```diff
@@ -1,61 +1,62 @@
 import * as webnative from 'webnative'
-import { setup } from 'webnative'
 
-import { filesystemStore, sessionStore } from '../stores'
+import { filesystemStore, programStore, sessionStore } from '../stores'
 import { getBackupStatus, type BackupStatus } from '$lib/auth/backup'
 
-// TODO: Add a flag or script to turn debugging on/off
-setup.debug({ enabled: false })
 
 export const initialize = async (): Promise<void> => {
   try {
     let backupStatus: BackupStatus = null
 
-    const state: webnative.AppState = await webnative.app({ useWnfs: true })
+    const program: webnative.Program = await webnative.program({
+      id: { creator: "Fission", name: "WAT" },
```

Comment:
`id` in full-stack web frameworks is often used to represent a numeric, unique identifier. If it's used in the context of a database, it may also be auto-incrementing and sequential.

We discussed perhaps changing this to `info` or `programId` to distinguish that this identifier is non-numeric and represents registry-like information about the program.

Reply:
I opted for `tag` instead. Let me know if that works for everyone. Could also do `label` or `programId` (although less a fan of programId, single words 👍).

I said I was going to put it as the first parameter in the `program` function (as opposed to the config object). But that turned out to not be as "clean" as I thought it would be. Because that would mean an extra parameter for all the other functions too.